src/pkg/utils/sbom.go

```diff
@@ -104,7 +115,7 @@ func HandleSBOM(extractSBOM bool, SBOMArtifactPathMap map[string]string, bundleN
 		if err != nil {
 			return warns, err
 		}
-	} else if len(SBOMArtifactPathMap) > 0 {
+	} else if len(SBOMArtifactPathMap) > 0 && sbomArtifactCount > 0 {
```

Comment:
nit: `sbomArtifactCount` cannot be greater than `0` if `len(SBOMArtifactPathMap)` is not also, so we could simplify this to just `else if sbomArtifactCount > 0` yeah?